src/item/Armor.php

```diff
@@ -69,6 +73,18 @@ public function getMaxStackSize() : int{
 		return 1;
 	}
 
+	public function getEquipSound() : ?Sound{
+		return $this->equipSound;
+	}
+
+	/**
+	 * @return $this
+	 */
+	public function setEquipSound(?Sound $equipSound) : self{
```

Comment:
This should be removed. We don't persist this information, so it's going to vanish after a restart.